src/components/profile/ProfileInformation.tsx

```diff
@@ -20,40 +24,56 @@ import { StatItem } from './../stats/StatBar'
 
 const ProfileInformation = ({ handleLogOut, handleEditProfile }: any) => {
   const [signedInWithGoogle, setSignedInWithGoogle] = useState<boolean>(false)
+  const [user, loading, error] = useAuthState(auth)
```

Comment:
Because this is a child component of `SignInTabs` which does include `useAuthState` none of these variables should be needed here.